envFiles/.env.devcontainer

```
@@ -1,5 +1,9 @@
 ########## talawa api ##########
-
+AWS_SECRET_ACCESS_KEY=your_aws_secret_access_key
+AWS_ACCESS_KEY_ID=your_aws_access_key_id
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AWS_ACCESS_KEY_ID key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)